### PR TITLE
Added tenor.co support to lib/modules/hosts.

### PR DESCRIPTION
**Pull request by SeanHayes**

### CODE REVIEW

File: lib/modules/hosts/tenor.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	name: 'tenor',
+	domains: ['tenor.co'],
+	logo: 'https://www.tenor.co/favicon.ico',
+	parseViewShortId: s => {
+		s = s.split('');
+		let n = 0;
+
+		s.forEach(c => {
+			n = (n * _alphabet.length) + _alphabetMap[c];
+		});
```
Comment (SeanHayes) [inline lib/modules/hosts/tenor.js]: 1. We have short URLs in the form "https://tenor.co/s2Vn.gif", where "s2Vn" is en encoded int. This function decodes that int ID.
2. Just rewrote it to use reduce.

**Comment by SeanHayes** [comment]
Any chance this could get merged?

**Comment by erikdesjardins** [comment]
Thanks!

Sorry about the delay

**Comment by SeanHayes** [comment]
No, thank you!

**Pull request merged**